smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/ProtocolGenerator.kt

```diff
@@ -133,7 +128,7 @@ interface ProtocolGenerator {
         HttpTraitResolver(ctx, defaultContentType)
 
     val operationMiddleware: OperationMiddleware
-    val httpProtocolCustomizable: HttpProtocolCustomizable
+    val customizations: HTTPProtocolCustomizable
```

Comment:
Reference renamed types, some properties moved to customizations instead of being inherited.